commentHandler.go

```diff
@@ -256,21 +263,78 @@ func manageLocking(req types.IssueCommentOuter, cmdType string) (string, error)
 	return buffer.String(), nil
 }
 
+func manageMilestone(req types.IssueCommentOuter, cmdType string, cmdValue string) (string, error) {
+
+	milestoneValue := cmdValue
+
+	var buffer bytes.Buffer
+
+	milestoneAction := strings.Replace(strings.ToLower(cmdType), "milestone", "", 1)
+
+	buffer.WriteString(fmt.Sprintf("%s wants to %s milestone of '%s' on issue #%d \n", req.Comment.User.Login, milestoneAction, milestoneValue, req.Issue.Number))
+
+	client, ctx := makeClient(req.Installation.ID)
```

Comment:
move this line before the line with `theMilestones, _, milErr := client.Issues.`